pkg/gui/gui.go

```diff
@@ -638,6 +638,9 @@ func (gui *Gui) Run(startArgs appTypes.StartArgs) error {
 
 	gui.g.ShowListFooter = userConfig.Gui.ShowListFooter
 
+	gui.c.UserConfig.Gui.ShowPanelJumps = userConfig.Gui.ShowPanelJumps
+	gui.c.UserConfig.Keybinding.Universal.JumpToBlock = userConfig.Keybinding.Universal.JumpToBlock
+
```

Comment:
These look like no-ops to me (assigning values to themselves; `gui.UserConfig` and `gui.c.UserConfig` is the same instance). You probably want to simply drop this hunk.

Reply:
Yeah this felt odd to me, I was just trying to replicate what the surrounding code did (sorry `go` newbie here).